src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let score = 0;
+
+  return (x) => (y) => {
+    score++;
+
+    if (score > 3 && score % 2 === 0) {
+      return 'Bzzz... Error!';
+    }
+
+    return x + y;
+  };
```

Comment:
This version is much readable.
```suggestion
  return (x) => {
    return (y) => {
      score++;
  
      if (score > 3 && score % 2 === 0) {
        return 'Bzzz... Error!';
      }
  
      return x + y;
    };
  };
```